.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.m4 -text eol=lf
+*.ac -text eol=lf
```

Comment:
I don't think `eol` has any effect without `text`, so it should either be `-text` *or* `text eol=lf`

Reply:
Ok I'll try that

Reply:
After removing the dashes, workflow still working for me